### PR TITLE
[Service] NIT followups to `AddService` in #316

### PR DESCRIPTION
**Pull request by Olshansk**
## Summary

NIT followups to #316 added in a separate PR to avoid blocking the base PR.

## Type of change

Select one or more:

- [ ] New feature, functionality or library
- [ ] Bug fix
- [x] Code health or cleanup
- [ ] Documentation
- [ ] Other (specify)

## Testing

- [ ] **Run all unit tests**: `make go_develop_and_test`
- [ ] **Run E2E tests locally**: `make test_e2e`
- [ ] **Run E2E tests on DevNet**: Add the `devnet-test-e2e` label to the PR. This is VERY expensive, only do it after all the reviews are complete.

## Sanity Checklist

- [ ] I have tested my changes using the available tooling
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, updated documentation and left TODOs throughout the codebase

### CODE REVIEW

File: x/service/keeper/service_test.go [hunk 1]
```diff
@@ -7,14 +7,13 @@ import (
...
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
-	"github.com/stretchr/testify/require"
-
 	"github.com/pokt-network/poktroll/cmd/pocketd/cmd"
 	keepertest "github.com/pokt-network/poktroll/testutil/keeper"
 	"github.com/pokt-network/poktroll/testutil/nullify"
 	"github.com/pokt-network/poktroll/x/service/keeper"
 	"github.com/pokt-network/poktroll/x/service/types"
 	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+	"github.com/stretchr/testify/require"
```
[the remark this answers is not in the document]
Comment (Olshansk) [reply x/service/keeper/service_test.go:16]: Do you have any idea why this doesn't format it correctly?

Screenshot 2024-01-19 at 2 57 54 PM

**Comment by Olshansk** [comment]
@h5law Fixed the imports and will merge this in after E2E tests pass. Plz see my comment in https://github.com/pokt-network/poktroll/pull/338#discussion_r1459666500, since I'm trying to understand what formatted you use.

File: x/service/keeper/service_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (h5law) [reply x/service/keeper/service_test.go]: https://github.com/incu6us/goimports-reviser

https://github.com/golang/vscode-go/wiki/features#format-and-organize-imports

https://github.com/golang/vscode-go/wiki/advanced#formatting-code-and-organizing-imports

https://github.com/pokt-network/poktroll/blob/943669e0beaf011493273c773ee288d22684702f/.golangci.yml#L30

These links should help, I don't know exactly as I don't use vscode - but the last link should help as a reference on the 2nd link.

[the remark this answers is not in the document]
Comment (h5law) [reply x/service/keeper/service_test.go]: goimports != goimports-reviser you want a final group which is prefixed with our repo name. Pretty sure there is a goimports-reviser plugin maybe for vscode but whatever way you find to make it work - document it somewhere so we can make a configs section

[the remark this answers is not in the document]
Comment (h5law) [reply x/service/keeper/service_test.go]: https://chat.openai.com/share/d2590505-31e2-4e34-be93-8e4fabf8d872

[the remark this answers is not in the document]
Comment (Olshansk) [reply x/service/keeper/service_test.go]: https://github.com/incu6us/goimports-reviser/issues/147 🤷

**Pull request merged**